prime-angular-frontend/src/app/lib/modules/ngx-busy/busy-overlay-message/busy-overlay-message.component.html

```diff
@@ -1,8 +1,11 @@
 <div class="ng-busy-default-wrapper">
+  <div class="align-center">
+    <div class="center mb-5">
+      <mat-spinner></mat-spinner>
+    </div>
 
-  <!--
-  // NOTE: Already obvious to user that load is occurring this seems
-  // annoying, but left in case it becomes required
-  -->
-
+    <div class="center delay">
+      <h3>{{message}}</h3>
+    </div>
```

Comment:
The message `Wait patiently, the submission can take up to a minute` seems to be on top of existing text in the page and might be causing readability issue if overlapping.

![image](https://user-images.githubusercontent.com/58282608/152068947-af2c0f0d-60e2-4359-be16-349582604d83.png)

Reply:
I made the backdrop more opaque if a message is provided.